playground/src/components/Editor.svelte

```diff
@@ -72,7 +79,32 @@
 
   $effect(() => {
     if (!editor) return;
-    editor.addCommand(monaco.KeyCode.Enter, () => onEnter?.());
+
+    let isAutoCompleteOpenCtx = editor.createContextKey<boolean>(
+      IS_AUTOCOMPLETE_OPEN_CTX_KEY,
+      false,
+    );
+
+    const suggestController = editor.getContribution(
+      "editor.contrib.suggestController",
+    ) as any;
```

Comment:
isnt there any better option the any?

Reply:
Just to rewrite the type, this type does not exist in the published code of both `monaco-editor` and `monaco-editor-core`